packages/@coorpacademy-components/src/template/my-learning/index.js

```diff
@@ -206,16 +212,16 @@ const MyLearning = (props, context) => {
       icon: {iconName: 'book-open-cover', backgroundColor: '#D9F4F7'}
     },
     {
-      title: translate('skill_chart_side_panel_questions_answered'),
-      value: `${questionsAnsweredData}`,
+      title: translate('skill_chart_side_panel_learning_hours'),
+      value: handleLearningTimeFormat(learningHoursData),
       legend: skillChartPaneLegends,
-      icon: {iconName: 'circle-question', backgroundColor: '#FFDCD1'}
+      icon: {iconName: 'clock', backgroundColor: '#FAD6DE'}
     },
     {
-      title: translate('skill_chart_side_panel_learning_hours'),
-      value: `${learningHoursData}`,
+      title: translate('skill_chart_side_panel_questions_answered'),
+      value: `${questionsAnsweredData}`,
```

Comment:
and, on second thought,  you can in fact adapt the props name now. We'll update the mooc as we bump the components